ui/client/documents/index.js

```diff
@@ -452,21 +452,23 @@ const ViewDocumentsGrid = withStyles((theme) => ({
         Promise.all(allDocPromises).then(() => {
           setDocParagraphResults(allParagraphs);
 
-          // fetch all highlights for results
-          const matches = map(allParagraphs, 'text');
-          const query = searchTerm;
-
-          let url = `/api/dojo/paragraphs/highlight`;
-
-          return axios.post(url, {
-            query,
-            matches
-          })
-            .then((response) => {
-              setHighlights(response.data.highlights)
+          if (process.env.DISABLE_SEMANTIC_HIGHLIGHT !== "true") {
```

Comment:
Only accepts "true" to disable, any other value will not disable semantic highlight, as requested.